src/components/chip/CdrChipGroup.vue

```diff
@@ -37,7 +37,7 @@ export default defineComponent({
 
     onMounted(() => {
       chips.value = Array.prototype.filter.call(chipsEl.value.children,
-        (chip) => !(chip.getAttribute('disabled') || chip.getAttribute('aria-disabled')));
+        (chip) => !(chip.getAttribute('disabled')  === '' || chip.getAttribute('aria-disabled')));
```

Comment:
If an element has a disabled attribute, it returns an empty string, not a boolean value which is weird